run-checks

```diff
@@ -393,6 +393,25 @@ if [ "$STATIC" = 1 ]; then
       echo "$got"
       exit 1
     fi
+
+    if [ -z "${SKIP_GOLANG_CI_LINT-}" ]; then
+
+        echo "Checking installation of golangci-lint"
+        if ! command -v golangci-lint > /dev/null ; then
+            echo "ERROR: Cannot find golangci-lint. You need to first install the golangci-lint snap"
+            exit 1
+        fi
+        if snap refresh --list | grep golangci-lint; then
+            echo "WARNING: your golangci-lint snap is out of date. The CI will install a fresh version, which may differ from yours."
+        fi
```

Comment:
I think checking that they're tracking golangci-lint from the same channel (`latest/stable`) could be worthwhile. Or at least from `latest/*`. It's probably safe to assume that `latest/{candidate,beta,edge}` is at least as strict as `latest/stable`.

Reply:
I think it's probably worth printing a warning if they're running a channel other than `latest/stable` (or whatever the CI is declared to use), at the very least.